File: cfgs/pythoncfgs/WRITE_A_C_PROGRAM_TO_CALCULATE_POWXN.py
Nodes
1;conditional;foo_1 == 0;8;
2;conditional;fun_1(foo_1 % 2) == 0;9;
3;statement;return foo_1 * fun_2(foo_2, fun_2(foo_3 / 2)) * fun_4(foo_4, fun_4(foo_5 / 2));12;
4;statement;return fun_2(foo_1, fun_2(foo_2 / 2)) * fun_4(foo_3, fun_4(foo_4 / 2));10;
5;statement;return 1;8;
6;exit;;
Edges
0;2;5;
1;3;4;
2;6;
3;6;
4;6;
5;

File: cfgs/java_cfgs_foo/WRITE_A_C_PROGRAM_TO_CALCULATE_POWXN.java
Nodes
1;special;;;
2;conditional;foo_1 == 0;14;
3;statement;return 1;14;
4;exit;;;
5;conditional;(foo_1 % 2) == 0;15;
6;statement;return fun_1(foo_1, foo_2 / 2) * fun_2(foo_3, foo_4 / 2);15;
7;statement;return (foo_1 * fun_1(foo_2, foo_3 / 2)) * fun_2(foo_4, foo_5 / 2);16;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;7;
5;4;
6;4;
